coderd/notifications_test.go

```diff
@@ -60,4 +60,22 @@ func TestUpdateNotificationsSettings(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, expected, actual)
 	})
+
+	t.Run("Settings not modified", func(t *testing.T) {
+		t.Parallel()
+
+		client := coderdtest.New(t, nil)
+		_ = coderdtest.CreateFirstUser(t, client)
+
+		ctx := testutil.Context(t, testutil.WaitShort)
+
+		// given
+		expected := codersdk.NotificationsSettings{
+			NotifierPaused: false,
+		}
+
+		// then
+		err := client.PutNotificationsSettings(ctx, expected)
+		require.NoError(t, err)
```

Comment:
Sorry to keep riding this test.

The absence of an error isn't specific enough for this test, I feel.
To validate that this endpoint is idempotent, we should:

1. start from an empty state
2. change the state
3. validate the state
4. try change the state again
5. validate the state is unchanged from step 3

Reply:
Refactored 👍 Let me know your thoughts.

Reply:
Perfect!